app/routes/activities.js

```diff
@@ -49,7 +50,12 @@ module.exports = [{
         activityLabel: (await getActivityById(request.payload.activity)).label
       }
 
-      await sendActivityToAudit(payload, getCallingUser(request))
+      if (payload.activityLabel === 'Application pack' && payload.activityType === 'sent') {
```

Comment:
Is this 'if' statement going to grow with additional actions? If so, it would be better to do that logic in the cdoService, for example, or it could get messy in the route